src/components/Envelope.vue

```diff
@@ -1053,6 +1060,10 @@ export default {
 	display: flex;
 	width: 200px;
 }
+.line-two {
+	display: flex;
+	align-items: center;
+}
```

Comment:
```suggestion
```

Without those styles the preview texts work again and the original issue still seems to be fixed.

Reply:
this is needed for the list layout, fixing it now